CMakeLists.txt

```diff
@@ -35,7 +35,12 @@ endif()
 
 # LIBM {
 include(CheckLibraryExists)
-check_library_exists(m pow "" WITH_HDRHISTOGRAM)
+check_library_exists(m pow "" HAVE_LIBM)
```

Comment:
On MSVC we don't need to, and probably shouldnt, check for libm at all.
What do you think?

Reply:
@edenhill Thanks for your review! I will update this PR later.